extensions/ql-vscode/src/extension.ts

```diff
@@ -952,13 +952,9 @@ async function activateWithInstalledDistribution(
       token: CancellationToken
     ) => {
 
-      const input = [
-        variantAnalysisManager.queue.add(async () => {
-          await variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
-        })
-      ];
-
-      await Promise.all(input);
+      await variantAnalysisManager.queue.add(async () => {
+        await variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
+      });
```

Comment:
I initially experimented with:

```
await variantAnalysisManager.queue.add(variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
});
```

but `queue.add()` expects to be passed a function as a parameter, rather than a promise.